internal/web_server/web/handlers/quiz_pages_handlers.go

```diff
@@ -200,7 +201,11 @@ func (qph *QuizPagesHandler) getQuizSummary(c echo.Context) error {
 }
 
 func (qph *QuizPagesHandler) getScoreboard(c echo.Context) error {
-	return utils.Render(c, http.StatusOK, quiz_pages.Scoreboard())
+        rankings, err := userranking.GetRanking(qph.sharedData.DB)
+        if err != nil {
+                return echo.NewHTTPError(http.StatusInternalServerError, "Failed to get rankings")
```

Comment:
return the error by itself instead, it will turn into 500 and will be logged automatically by error handler

try to do custom http error only for client errors (4xx)